polyglot.js

```diff
@@ -201,7 +205,7 @@ class PolyGlot {
 
     renderChatMessage(message, html, data) {
         // html and data are swapped on 0.3.x in relation to other render<Application> hooks
-        if (message.data.type == CONST.CHAT_MESSAGE_TYPES.IC) {
+        if (message.data.type == CONST.CHAT_MESSAGE_TYPES.IC || (this.allowOOC && message.data.type < 5 && message.data.type > 0)) {
```

Comment:
This doesn't check that the message author is a GM, so if the option is enabled, it would transform any OOC message.

Reply:
This one does not check if the client that is rendering it is a GM, otherwise only GM would see their messages as the selected language, and the players would see them without any. I've fixed this in the latest commits, as those functions shouldn't check if the option to allow OOC is selected. Turning it off would show the previous messages as unrunified to the players.